clusters/threat-actor.json

```diff
@@ -644,14 +638,11 @@
           "https://www.secureworks.com/research/threat-profiles/bronze-export",
           "https://www.pwc.co.uk/cyber-security/assets/cyber-threats-2019-retrospect.pdf",
           "https://www.justice.gov/opa/pr/seven-international-cyber-defendants-including-apt41-actors-charged-connection-computer",
-          "https://assets.documentcloud.org/documents/7210602/FLASH-AC-000133-TT-Published.pdf"
+          "https://assets.documentcloud.org/documents/7210602/FLASH-AC-000133-TT-Published.pdf",
+          "https://www.cfr.org/cyber-operations/winnti-umbrella"
         ],
         "synonyms": [
           "Winnti Umbrella",
-          "Winnti Group",
-          "Suckfly",
```

Comment:
+1 for branching out Suckfly. No relation to APT41 or Axiom.

Reply:
Also, please merge `Suckfly` into `APT22` per https://www.secureworks.com/research/threat-profiles/bronze-olive